Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -126,7 +127,18 @@ int VertexLoaderX64::ReadVertex(OpArg data, u64 attribute, int format, int count
 	{
 		case 1: MOVSS(dest, coords); break;
 		case 2: MOVLPS(dest, coords); break;
-		case 3: MOVUPS(dest, coords); break;
+		case 3:
+			MOVUPS(dest, coords);
+			if (native_format == &m_native_vtx_decl.position)
+			{
+				CMP(32, R(count_reg), Imm8(3));
+				FixupBranch dont_store = J_CC(CC_A);
+				IMUL(64, scratch1, R(count_reg), Imm8(-3));
+				u64 offset = (u64)&VertexLoaderManager::position_cache[9];
+				MOVUPS(MScaled(scratch1, sizeof(float), offset), coords);
```

Comment:
I know, I just like it when Windows is the only failing build. It also prevents this from being merged, so that's good.